app_dev.yaml

```diff
@@ -37,14 +37,15 @@ handlers:
   static_dir: build
   secure: always
   expiration: "90d"
+  application_readable: true
```

Comment:
I believe we try to keep our static handlers and dynamic handlers separate (i.e. the application does't reference static data directly). Is this change actually needed, and if so, why?